jpeg/BUILDING.md

```diff
@@ -0,0 +1,32 @@
+# Instructions to build the static libraries
+
+## Current libraries version
+
+The current static libraries are version 2.1.5.1 (see [official binaries](https://sourceforge.net/projects/libjpeg-turbo/files/2.1.5.1/) and [Github Project Page](https://github.com/libjpeg-turbo/libjpeg-turbo)).
+## Darwin and Linux
```

Comment:
As this only works for x86_64 linux (there is now brew for arm64 linux), I would suggest changing instructions to use the official builds for linux AND windows, and only use brew for mac.) Assuming that works.

Reply:
Extracting the libraries and header files from the official binaries doesn't work on linux x86_64. I haven't tested it on arm64 linux (but works when built from the brew bottles). So I would just add a link to the brew jpeg-turbo URL on this introductory paragraph if it's ok?

Reply:
Okay, if the official ones don't work, that's fine then. It's just a shame there's now brew bottle for linux/arm64 and thus someone has to build it manually. Oh well, at least these shouldn't need to update very often.